java/arcs/core/data/testdata/WriterReaderExample.arcs

```diff
@@ -44,7 +44,7 @@ particle ReadWriteReferences
   arcId referencesArcId
 recipe ReferencesRecipe
   things: create persistent 'my-refs-id'
-  thing: create persistent 'my-ref-id'
+  thing: create 'my-ref-id' @ttl(1d)
```

Comment:
Confirmed by the goldens, this generates an in memory db. Is this `queryable`?  
If so, what distinguishes the default capability from a `queryable` modifier?

Reply:
After reading more of the PR: when there's a TTL, then handles are promoted to be queryable.

Reply:
I think this nuance should be documented -- or, we should enforce that users add the `queryable` modifier when using TTLs. WDYT?

Reply:
Lol, I just read your description! Would have some me some time. :)